view/frontend/web/js/productlabel-position.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade this module to newer
+ * versions in the future.
+ *
+ * @category  Smile
+ * @package   Smile\ProductLabel
+ * @author    Houda EL RHOZLANE <houda.elrhozlane@smile.fr>
+ * @copyright 2019 Smile
+ * @license   Open Software License ("OSL") v. 3.0
+ */
+
+define([
+        "jquery"
+    ], function($){
+
+        var widthOfParent = $("#labels").parent().width();
+        var heightOfParent = $("#labels").parent().height() - 44;
```

Comment:
why "-44", is it your favorite number ? :)

I guess it's working on the default Luma theme on desktop, but if you reduce the screen or simulate mobile view, does it continue to work properly ? :)

Reply:
lol yes it works